prod-refractr.yml

```diff
@@ -832,12 +824,116 @@ refracts:
+    - if: '$request_uri ~ "(?i)^/pair/?$"'
+      ^: 'accounts.firefox.com/pair/'
+    - if: '$request_uri ~ "(?i)^/(join|rejoindre)/?$"'
+      ^: 'www.mozilla.org/firefox/accounts/?redirect_source=join'
+    - if: '$request_uri ~ "(?i)^/(privacy|privatsphaere)/?$"'
+      ^: 'www.mozilla.org/firefox/privacy/products/?redirect_source=firefox-com'
+    - if: '$request_uri ~ "(?i)^/viepriv(e|é)e/?$"'
+      ^: 'www.mozilla.org/firefox/privacy/products/?redirect_source=firefox-com'
+    - if: '$request_uri ~ "(?i)^/nightly/?$"'
+      ^: 'www.mozilla.org/en-US/firefox/channel/desktop/#nightly'
+    - if: '$request_uri ~ "(?i)^/armchair/?$"'
+      ^: 'www.mozilla.org/firefox/unfck/?redirect_source=firefox-com&utm_medium=audio&utm_source=armchair&utm_campaign=unfck&utm_content=podcast'
+    - if: '$request_uri ~ "(?i)^/jvn/?$"'
+      ^: 'www.mozilla.org/firefox/unfck/?redirect_source=firefox-com&utm_medium=audio&utm_source=jvn&utm_campaign=unfck&utm_content=podcast'
+    - if: '$request_uri ~ "(?i)^/literally/?$"'
+      ^: 'www.mozilla.org/firefox/unfck/?redirect_source=firefox-com&utm_medium=audio&utm_source=literally&utm_campaign=unfck&utm_content=podcast'
+    - if: '$request_uri ~ "(?i)^/pivot/?$"'
+      ^: 'www.mozilla.org/firefox/unfck/?redirect_source=firefox-com&utm_medium=audio&utm_source=pivot&utm_campaign=unfck&utm_content=podcast'
+    - if: '$request_uri ~ "(?i)^/podsave/?$"'
+      ^: 'www.mozilla.org/firefox/unfck/?redirect_source=firefox-com&utm_medium=audio&utm_source=podsave&utm_campaign=unfck&utm_content=podcast'
+    - if: '$request_uri ~ "(?i)^/smartless/?$"'
+      ^: 'www.mozilla.org/firefox/unfck/?redirect_source=firefox-com&utm_medium=audio&utm_source=smartless&utm_campaign=unfck&utm_content=podcast'
+    - if: '$request_uri ~ "(?i)^/thedaily/?$"'
+      ^: 'www.mozilla.org/firefox/unfck/?redirect_source=firefox-com&utm_medium=audio&utm_source=thedaily&utm_campaign=unfck&utm_content=podcast'
+    - if: '$request_uri ~ "(?i)^/ezra/?$"'
+      ^: 'www.mozilla.org/firefox/unfck/?redirect_source=firefox-com&utm_medium=audio&utm_source=ezra&utm_campaign=unfck&utm_content=podcast'
+    - if: '$request_uri ~ "(?i)^/explained/?$"'
+      ^: 'www.mozilla.org/firefox/unfck/?redirect_source=firefox-com&utm_medium=audio&utm_source=explained&utm_campaign=unfck&utm_content=podcast'
+    - if: '$request_uri ~ "(?i)^/wtf/?$"'
+      ^: 'www.mozilla.org/firefox/unfck/?redirect_source=firefox-com&utm_medium=audio&utm_source=wtf&utm_campaign=unfck&utm_content=podcast'
+    - if: '$request_uri ~ "(?i)^/liebe/?$"'
+      ^: 'www.mozilla.org/firefox/unfck/?redirect_source=firefox-com&utm_medium=audio&utm_source=prosieben&utm_campaign=unfck&utm_content=podcast'
+    - if: '$request_uri ~ "(?i)^/(unfu?ck|love|rendonslenetplusnet)/?$"'
+      ^: 'www.mozilla.org/firefox/unfck/?redirect_source=firefox-com&utm_medium=referral&utm_source=firefox.com&utm_campaign=unfck'
```

Comment:
Only difference I see here is passing the `utm` if it's already set.

Reply:
yeah, talked with pmac and we decided it was ok to drop the utm stuff.  This campaign is finished, so losing some utm fidelity probably won't matter at all.